blueocean-bitbucket-pipeline/src/main/java/io/jenkins/blueocean/blueocean_bitbucket_pipeline/BitbucketApiFactory.java

```diff
@@ -14,7 +14,7 @@
  *
  * @author Vivek Pandey
  */
-public abstract class BitbucketApiFactory extends ExtensionPoint{
+public abstract class BitbucketApiFactory implements ExtensionPoint {
```

Comment:
Wow! Thanks @batmat, I think luckily it work because lookup is not done using ExtensionPoint, It’s done using BitbucketApiFactory.